src/xunit.analyzers/xunit.analyzers.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageTargetFallback>portable-net45+win8</PackageTargetFallback>
     <RootNamespace>Xunit.Analyzers</RootNamespace>
     <TargetFramework>netstandard1.1</TargetFramework>
+    <DebugType>full</DebugType>
```

Comment:
@marcind, I was having a really hard time debugging the VSIX because the breakpoints weren't being hit. I eventually realized the fix was to add this and then add the `xunit.analyzers\bin\Debug\netstandard1.1\` dir to the PDB search path.